subprojects/platform-base/build.gradle.kts

```diff
@@ -42,3 +42,5 @@ classycle {
 }
 
 integTest.usesJavadocCodeSnippets.set(true)
+
+description = """Provides general purpose types for runtime and language support."""
```

Comment:
I'm struggling to come up with a short description of this project.

Reply:
I'm not really familiar with this, but it looks like a whole lotta software model-related stuff. `LifecycleBasePlugin` `ProjectLayout` are the only things I recognized.